packages/cf.js/src/types/structs.ts

```diff
@@ -0,0 +1,28 @@
+import { ABIEncoding, Address } from "./simple-types";
+import { BigNumber } from "ethers/utils";
+
+export interface AppDefinition {
```

Comment:
Do we still need this?

Reply:
We're still using it throughout cf.js, including sometimes referencing `appDefinition.address`. We couldn't remove it without breaking that code, and there's no obvious replacement for it either.